lib/web/scripts/node-join/install.sh

```diff
@@ -885,7 +885,7 @@ install_from_repo() {
         # See: https://github.com/gravitational/teleport/issues/22581
         yum --disablerepo="*" --enablerepo="teleport" clean metadata
         
-        yum install -y ${TELEPORT_PACKAGE_NAME}
+        yum install -y ${TELEPORT_PACKAGE_NAME}-${TELEPORT_VERSION}-1
```

Comment:
This will break on versions with suffixes (e.g. `alpha.1`).  Yum version strings have `-` replaced with `_`, meaning that `13.0.0-alpha.1` becomes `13.0.0_alpha.1`.  Previously I have used bash substitution to handle this in scripts and it has worked.  Ex:

```bash
yum_target_version="${target_version//-/_}" # expansion replaces all instances of '-' with '_'
```

Additionally, my understanding is that it is not standard practice to hard-code a yum package release number (the `-1` suffix).  I've never encountered a scenario where I actually needed to specify the package release number.  If there is some reason it needs to be specified here, I'd include comments explaining why.  Otherwise, I'd omit it.

Reply:
I'll add that
I suspect only standard and prod-ready versions will be expose on our official repo, but adding a check here should only make this more resilient

I'm also removing the packager version, I was not aware that yum would accept the version without the package release number.

I'll do some manual tests on the most common distros and get back to you
Thank you for the review
:pray: